scene/animation/animation_tree.cpp

```diff
@@ -1009,7 +1032,6 @@ void AnimationTree::_process_graph(float p_delta) {
 							t->value = bezier;
 							t->process_pass = process_pass;
 						}
-
 						t->value = Math::lerp(t->value, bezier, blend);
```

Comment:
```suggestion

						t->value = Math::lerp(t->value, bezier, blend);
```
Unrelated whitespace chance, restore